### PR TITLE
Update OCI Static yml

### PR DESCRIPTION
**Pull request by djnakabaale**
- add usage_consumed_quantity to oci static yaml
- update nise version

### CODE REVIEW

File: nise/generators/oci/oci_network_generator.py [hunk 1]
```diff
@@ -34,7 +34,11 @@ def __init__(self, start_date, end_date, currency, attributes=None):
         self.usage_consumed_quant_units = "BYTES"
         self.usage_consumed_quant_measure = "DATA_TRASFERED"
         self.usage_product_resource = "PIC_COMPUTE_OUTBOUND_DATA_TRANSFER_ZONE1"
-        self.usage_consumed_quantity = self.fake.pyint(max_value=153306)
```
Comment (myersCody) [inline nise/generators/oci/oci_network_generator.py:37]: Did these max values matter before? or were they just random?

Comment (djnakabaale) [reply nise/generators/oci/oci_network_generator.py:37]: those are random

File: nise/generators/oci/oci_network_generator.py [hunk 2]
```diff
@@ -34,7 +34,11 @@ def __init__(self, start_date, end_date, currency, attributes=None):
...
+        self.usage_consumed_quantity = (
+            attributes.get("consumed_quantity")
+            if attributes and attributes.get("consumed_quantity")
+            else self.fake.pyint(min_value=3600000, max_value=86400000)
```
Comment (pedrolp85) [inline nise/generators/oci/oci_network_generator.py:40]: Does network usage use miliseconds too?

File: nise/generators/oci/oci_database_generator.py [hunk 3]
```diff
@@ -57,7 +57,11 @@ def __init__(self, start_date, end_date, currency, attributes=None):
         self.usage_consumed_quant_units = self.select_db_resource.get("usage_consumed_quant_units")
         self.usage_consumed_quant_measure = self.select_db_resource.get("usage_consumed_quant_measure")
         self.usage_product_resource = self.select_db_resource.get("usage_product_resource")
-        self.usage_consumed_quantity = self.fake.pyint(max_value=167674224)
+        self.usage_consumed_quantity = (
+            attributes.get("consumed_quantity")
+            if attributes and attributes.get("consumed_quantity")
+            else self.fake.pyint(min_value=3600000, max_value=86400000)
```
Comment (pedrolp85) [inline nise/generators/oci/oci_database_generator.py:63]: Does database usage use miliseconds too?

Comment (djnakabaale) [reply nise/generators/oci/oci_database_generator.py]: those are all random values

**Review by myersCody — APPROVED** [review]
LGTM

**Pull request merged**